sdk/iot/Azure.Iot.Hub.Service/src/Customized/JobsRestClient.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service
+{
+    /// <summary>
+    /// Override to return string instead of object.
+    /// </summary>
+    internal partial class JobsRestClient
+    {
+        internal Response<string> CancelImportExportJob(string id, CancellationToken cancellationToken = default)
```

Comment:
No async version of this method?

Reply:
Looks like the async version exists. I will add it to our overrides as well.